bootstrap-sass/src/test/java/de/agilecoders/wicket/sass/BootstrapSassTest.java

```diff
@@ -72,8 +72,9 @@ public void importWebContext() throws Exception {
 
     /**
      * Tests with {@link ContextRelativeSassResourceReference}
-     *
+     * <p>
```

Comment:
Sorry to chime in here. Why is this change? Checkstyle nagging about trailing whitespace? (That rule can be relaxed with a custom one.)

If so, is it part of the build? (From the looks it seems disabled.) Or is this a IDE-local check which wasn't necessary from the projects build rules?

Reply:
Yes, Idea autoformatted it on save. I missed it before submitting PR.

Reply:
I dunno about a preferred style for this project. I am just a user watching the project.

I've seen that "<p>" in other projects just to calm down Checkstyles default rule for dangling whitespace. So no issue on my side. Was just interested.